_posts/2020-18-06-Triplestores-Onotologies-Reasoning.md

```diff
@@ -15,22 +15,29 @@ Triplestores and Ontologies in depth with a focus on the IrishGen
 dataset.  If the reader is new to Semantic Web/Linked Data then
 beginning with ["IrishGen: RDF, Linked Data, and The Semantic Web"]({%
 post_url 2020-06-17-IrishGen-RDF-Linked-Data-Semantic-Web %}) will
-orient the reader before approaching this set of topics.
+orient the reader before approaching this set of topics.  This guide
+is a theoretical examination of Triplestores and Ontologies.  If the
+reader is looking for a more practical introduction, Eystein
+Thanisch's ["Basic Querying and more on the Cíannachta Glinne
+Geimen"]({% post_url 2020-06-20-Basic-Querying %}) which gives the
```

Comment:
Thanks for the shout out! However, the title has now changed to "Some Examples of Querying and More on the Cíannachta Glinne Geimen" / 2020-06-20-Some-Examples-of-Querying.md .